lib/db/schema/patch-073-074.sql

```diff
@@ -0,0 +1,179 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE TABLE IF NOT EXISTS devicesCapabilities (
+  uid BINARY(16) NOT NULL,
+  deviceId BINARY(16) NOT NULL,
+  capability TINYINT UNSIGNED NOT NULL,
+  PRIMARY KEY (uid, deviceId, capability),
+  FOREIGN KEY (uid, deviceId) REFERENCES devices(uid, id) ON DELETE CASCADE
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `addCapability_1` (
+  IN `inUid` BINARY(16),
+  IN `inDeviceId` BINARY(16),
+  IN `inCapability` TINYINT UNSIGNED
+)
+BEGIN
+  INSERT INTO devicesCapabilities(
```

Comment:
*Even more* bike-shedding (sorry)... are we sure about the plural on the `devices` part here? `deviceCapabilities` sounds more natural to me.